javascript/frameworks/xsjs/lib/advanced_security/javascript/frameworks/xsjs/RemoteFlowSources.qll

```diff
@@ -1,30 +0,0 @@
-import javascript
-import DataFlow
-import advanced_security.javascript.frameworks.xsjs.AsyncXSJS
-
-class XSJSZipClassInstantiation extends NewNode {
```

Comment:
Do we have this node in the models file?

Reply:
This definition ramified into [XSJSZipInstance](https://github.com/advanced-security/codeql-sap-js/blob/3dc4f40f100500d43640d749b4523d98c6f2de7d/javascript/frameworks/xsjs/lib/advanced_security/javascript/frameworks/xsjs/AsyncXSJS.qll#L218-L222) and a more specific class [XSJSZipInstanceDependingOnRemoteFlowSource](https://github.com/advanced-security/codeql-sap-js/blob/3dc4f40f100500d43640d749b4523d98c6f2de7d/javascript/frameworks/xsjs/lib/advanced_security/javascript/frameworks/xsjs/XSJSZipSlipQuery.qll#L10-L18).